pkg/pillar/cmd/downloader/syncop.go

```diff
@@ -117,6 +117,7 @@ func handleSyncOp(ctx *downloaderContext, key string,
 		// pass in the config.Name instead of 'filename' which
 		// does not contain the prefix of the relative path with '/'s
 		remoteName = config.Name
+		cleanOnError = false
```

Comment:
Maybe here (and for the S3 case above) it would make sense to add a comment
// Azure zedUpload can resume from a previous partial download.

Or maybe it makes sense to rename the boolean to supportsPartialDownload?